public/src/main/java/com/revenuecat/purchases/models/PurchaseOption.kt

```diff
@@ -31,4 +31,6 @@ interface PurchaseOption : Parcelable {
      */
     val isBasePlan: Boolean
         get() = pricingPhases.size == 1
+
+    val purchasingData: PurchasingData
```

Comment:
Hmm this is also public... Maybe we could mark this property as `internal`? Basically I don't think devs should know of the existence of `PurchasingData` at all.

Reply:
I tried this but I kept getting yelled at for `internal` not being able to be applied in an interface 🤷‍♂️